fv/iptables_lock_test.go

```diff
@@ -115,7 +115,7 @@ var _ = Describe("with running container", func() {
 		It("iptables should succeed in getting the lock after 3s", func() {
 			iptCmd := cmdInContainer("iptables", "-w", "3", "-A", "FORWARD")
 			out, err := iptCmd.CombinedOutput()
-			Expect(string(out)).To(ContainSubstring("Another app is currently holding the xtables lock"))
+			log.Printf("iptables output='%s'", out)
```

Comment:
Since we're using Logrus, probably best to use `Infof` to avoid confusion (Logrus' Printf behaves differently to the built in one)